packages/calcite-components/src/components/tile-select-group/tile-select-group.tsx

```diff
@@ -9,6 +9,7 @@ import {
 import { TileSelectGroupLayout } from "./interfaces";
 
 /**
+ * @deprecated  Use the `calcite-tile-group` component instead.
```

Comment:
nitpick:

It looks like there's two spaces between the `@deprecated` and `"Use"`.